pallets/api/src/messaging/test_utils.rs

```diff
@@ -14,9 +14,9 @@ pub fn ismp_get_request() -> GetRequest {
 		dest: StateMachine::Polkadot(2001),
 		nonce: 100u64,
 		from: vec![],
-		keys: vec![vec![]],
+		keys: vec![[1u8].repeat(1000)],
 		height: 1,
-		context: vec![],
+		context: [1u8].repeat(1000),
```

Comment:
seems like this might be better to make these into a parameter that is passed by the benchmark function (i.e., what is x currently equal to).

Reply:
great idea.